internal/app/cache/cache.go

```diff
@@ -236,10 +240,33 @@ func (c *cache) DeleteRequest(key string, req transport.Request) error {
 		"node_id", req.GetNodeID(),
 		"request_type", req.GetTypeURL(),
 	).Debug(context.Background(), "request deleted")
-	metrics.CacheAddRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteSuccess).Inc(1)
+	metrics.CacheDeleteRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteSuccess).Inc(1)
 	return nil
 }
 
+func (c *cache) DeleteKey(key string) (Resource, marshallable.Error) {
```

Comment:
The `marshallable.Error` should be used at the periphery of webapi calls. The rest of the code should return `error`.